### PR TITLE
[ServiceBus]: SDK for 2021-11-01 api version

### PR DESCRIPTION
**Pull request by damodaravadhani**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: eng/mgmt/mgmtmetadata/servicebus_resource-manager.txt [hunk 1]
```diff
@@ -3,12 +3,13 @@ AutoRest installed successfully.
...
+2022-06-06 06:37:03 UTC
 Azure-rest-api-specs repository information
 GitHub fork: Azure
 Branch:      main
-Commit:      9b489ff217acd3bd6d62b2932e42d3c08ed4d08c
+Commit:      475e39b72894a8bdef428a472bc6d661c50032eb
 AutoRest information
 Requested version: v2
-Bootstrapper version:    autorest@1.9.1
+Bootstrapper version:    autorest@3.4.2
```
Comment (ArthurMa1978) [inline eng/mgmt/mgmtmetadata/servicebus_resource-manager.txt:15]: autorest@3.4.2 is for track2, and here this pr is track 1 and which is being deprecated soon, here already have track 2 https://www.nuget.org/packages/Azure.ResourceManager.ServiceBus/1.0.0-beta.4 released

Comment (damodaravadhani) [reply eng/mgmt/mgmtmetadata/servicebus_resource-manager.txt:15]: @ArthurMa1978 , this is a management sdk, and we have been using this version of autorest for sometime now. We have used the same autorest version here as well https://github.com/Azure/azure-sdk-for-net/pull/28804. I also understand that using track 2 sdk can cause a lot of breaking changes. If we are generating the wrong way, could you please give me the command that uses the right track version. I am not sure I fully understand what you said. Thanks !

File: sdk/servicebus/Microsoft.Azure.Management.ServiceBus/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure ServiceBus management functions for managing the Microsoft Azure ServiceBus service.")]
 
 [assembly: AssemblyVersion("2.0.0.0")]
```
Comment (archerzz) [inline sdk/servicebus/Microsoft.Azure.Management.ServiceBus/src/Properties/AssemblyInfo.cs:10]: You need to bump the `AsemblyVersion` if it's just a major upgrade. See the [guide here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

> AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed

**Review by archerzz — CHANGES_REQUESTED** [review]
Make sure you update the following files, according to [the guide](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

- `AzSDK.RP.Props`

**Comment by damodaravadhani** [comment]
Hey @archerzz , could you please merge the PR if there aren't any additional concerns ?

File: sdk/servicebus/Microsoft.Azure.Management.ServiceBus/src/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure ServiceBus Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure ServiceBus management functions for managing the Microsoft Azure ServiceBus service.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("3.0.0.0")]
+[assembly: AssemblyVersion("3.0.0.0")]
```
Comment (archerzz) [inline sdk/servicebus/Microsoft.Azure.Management.ServiceBus/src/Properties/AssemblyInfo.cs:10]: I think both versions should be changed to `4.0.0.0`. See the [guide here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

> AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed

**Review by archerzz — CHANGES_REQUESTED** [review]
@damodaravadhani Please fix the inconsistency in the versions between `AssemblyInfo.cs` and the `.csproj` files. Thanks.

**Comment by damodaravadhani** [comment]
@archerzz , I have fixed the assembly version mismatch. Does this look good ?

**Comment by damodaravadhani** [comment]
@archerzz , can we merge this PR ?

**Review by archerzz — APPROVED** [review]

**Pull request merged**